lib/go-tc/deliveryservices.go

```diff
@@ -339,6 +339,13 @@ type DeliveryServiceRemovedFieldsV11 struct {
 	CacheURL *string `json:"cacheurl" db:"cacheurl"`
 }
 
+// RemoveLD1AndLD2 removes the Long Description 1 and Long Description 2 fields from a V 4.x DS, and returns the resulting struct.
+func (ds *DeliveryServiceV4) RemoveLD1AndLD2() DeliveryServiceV4 {
+	ds.LongDesc1 = nil
+	ds.LongDesc2 = nil
+	return *ds
+}
+
```

Comment:
I think this needs to be called in the upgrade method from v31 to v40

Reply:
That would break how his implementation works. It optionally inserts/updates those fields depending on which API version was requested, so we can't set them to nil in `UpgradeToV4` which is used in `createV31` and `updateV31`.

Reply:
Oh, right. DS hoops. Carry on :+1:

Reply:
I can't wait until everything has been collapsed up to v4.0 and we can get rid of all these hoops :)